internal/cli/ollama.go

```
@@ -0,0 +1,86 @@
+	base := strings.TrimRight(apiBaseURL, "/")
+	return strings.TrimSuffix(base, "/v1")
+}
+
+// ollamaDefaultModel asks the local daemon for its installed tags and returns
+// the first chat-capable id (falling back to the first tag when the payload
+// omits capabilities). It returns "" when the daemon is unreachable or the
+// catalog is empty, so callers keep their static default.
+func ollamaDefaultModel(apiBaseURL string) string {
+	client := &http.Client{Timeout: ollamaDiscoveryTimeout}
+	url := ollamaDaemonBaseURL(apiBaseURL) + "/api/tags"
+	resp, err := client.Get(url)
```

golangci-lint (2.12.2)

[error] (*net/http.Client).Get must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)